packages/opentelemetry-plugin-http/src/enums/AttributeNames.ts

```diff
@@ -15,17 +15,29 @@
  */
 
 /**
- * Attributes Names according to [OpenTelemetry attributes specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md#semantic-conventions)
+ * Attributes Names according to [OpenTelemetry attributes specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#common-attributes)
  */
 export enum AttributeNames {
```

Comment:
Looks `http.scheme` attribute is missing => https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#common-attributes

Reply:
Scheme is not required if you have `http.url` https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#http-client

Reply:
@mayurkale22
I should have spoken in the PR description. It is voluntary. In relation to @dyladan 's comment, I would add that it is redundant with component in the case of http / https plugins even if the attributes are not used for the same thing. WDYT ?

Reply:
SGTM, thanks for the explanation.

Reply:
> @mayurkale22
> I should have spoken in the PR description. It is voluntary. In relation to @dyladan 's comment, I would add that it is redundant with component in the case of http / https plugins even if the attributes are not used for the same thing. WDYT ?

The fact that it happens to match `component` in our case doesn't matter. This will be used by backend systems to classify, filter, and sort requests by end users. If a user says "only show me https calls", you can't use component for that because component may be `http` but it also may be some other http library.

Reply:
I'm agree. it's not for the same thing. It's just that there is a lot of new attributes (see screens in my previous comment) and  when I added `http.scheme`, it seemed not very important but I can add it if you want.

Reply:
It is required if URL is missing. If you have URL you don't need it